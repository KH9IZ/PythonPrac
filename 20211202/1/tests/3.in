class sqrt(metaclass=dump):
    def __repr__(self):
        return "I can take square roooooooot!! =)"
    def __call__(self, t):
        if issubclass(type(t), (int, float, complex)):
            return t ** 0.5
        return "No no no no no no STOP! What is this?!"
qq = sqrt()
print(qq)
print(f"{qq(2.2):.2f}")
print(qq('So...'))
