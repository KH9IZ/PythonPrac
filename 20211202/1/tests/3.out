__repr__: (), {}
I can take square roooooooot!! =)
__call__: (2.2,), {}
1.48
__call__: ('So...',), {}
No no no no no no STOP! What is this?!
